standalone/CMakeLists.txt

```
@@ -80,10 +80,14 @@ target_link_libraries(${PROJECT_NAME}_app
 ###########
 ## Tests ##
 ###########
-if (WIN32)
+if (BUILD_TESTING)
+
 include(CTest)
 
-if (BUILD_TESTING)
+# The standalone is included  with add_subdirectory from the top-level ROS CMakeLists.txt
+# However the tests are build separately in the top-level and thus should be ignored here.
+# If you don't know what ROS is, simply ignore this line.
+if(NOT DEFINED ENV{ROS_DISTRO})
```

```suggestion
if(NOT DEFINED ENV{catkin_FOUND})
```
This works also if you have ROS installed.